integrations/heap/lib/index.js

```diff
@@ -21,7 +21,9 @@ var toString = Object.prototype.toString; // in case this method has been overri
 var Heap = (module.exports = integration('Heap')
   .global('heap')
   .option('appId', '')
-  .tag('<script src="//cdn.heapanalytics.com/js/heap-{{ appId }}.js">'));
+  .option('hostname', 'cdn.heapanalytics.com')
```

Comment:
There seems to be some variation in how different integrations name this sort of parameter, so I wasn't sure of the best name. I'm happy to change to something else.